docker/iml-timer/iml-timer.conf

```diff
@@ -0,0 +1,20 @@
+location /timer/configure {
+    auth_request /auth;
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-Host $host;
+    proxy_set_header X-Forwarded-Server $host;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_pass http://iml-timer:8892/configure;
+}
+
+location /timer/unconfigure/ {
+    auth_request /auth;
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-Host $host;
+    proxy_set_header X-Forwarded-Server $host;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    rewrite ^/timer/unconfigure/(.*)$ /unconfigure/$1 break;
+    proxy_pass http://iml-timer:8892;
```

Comment:
With the above rewrite rule this looks similar to `proxy_pass http://iml-timer:8892/configure;`, no?

Reply:
Yeah this one is kind of tricky. I needed to use the rewrite rule in order to pass the id to the proxied server.

Reply:
I think a rewrite rule is not necessary according to https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/:

> If the URI is specified along with the address, it replaces the part of the request URI that matches the location parameter. For example, here the request with the /some/path/page.html URI will be proxied to http://www.example.com/link/page.html

Reply:
I believe this was my first approach actually and it didn't work. I can try changing it back to this and see if it works.

Reply:
I think it has to be `proxy_pass http://iml-timer:8892/unconfigure`

Reply:
And, maybe, it could be a single entry:

```
location /timer/ {
    auth_request /auth;
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-Host $host;
    proxy_set_header X-Forwarded-Server $host;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_pass http://iml-timer:8892/;
}
```

But I am not sure because I don't know the whole thing :)

Reply:
@johnsonw Have you tried leaving the trailing slash off the location directive? I.E.

`location /timer/unconfigure`

Also, you could probably match inside the location block itself to avoid the rewrite rule:

```
    location ~ /timer/unconfigure/(.+)$ {
        auth_request /auth;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Server $host;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_pass http://iml-timer:8892/unconfigure/$1;
    }
```

Reply:
The single block worked:
```
location /timer/ {
    auth_request /auth;
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-Host $host;
    proxy_set_header X-Forwarded-Server $host;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_pass http://iml-timer:8892/;
}
```

Configuring Timer:
![image](https://user-images.githubusercontent.com/1100657/74456229-7a0b4600-4e54-11ea-87a4-5e00f568355b.png)

Unconfiguring Timer:
![image](https://user-images.githubusercontent.com/1100657/74456317-a1faa980-4e54-11ea-952f-4f78c4f21c6c.png)

```
[root@iml-timer /]# systemctl list-timers
NEXT                         LEFT     LAST                         PASSED    UNIT                         ACTIVATES
Fri 2020-02-14 16:17:31 UTC  23h left Thu 2020-02-13 16:17:31 UTC  17min ago systemd-tmpfiles-clean.timer systemd-tmpfiles-clean.service

1 timers listed.
```
I will push a PR shortly with the update